versions/3.0.md

```diff
@@ -979,56 +998,75 @@ Field Name | Type | Description
 
 Field Pattern | Type | Description
 ---|:---:|---
+<a name="representations"></a>`*` | [Schema Object](#schemaObject)<span>&#124;</span> [Reference Object](#referenceObject) | A schema describing the response value for a specific `Content-Type`
 <a name="responseExtensions"></a>^x- | Any | Allows extensions to the OpenAPI Schema. The field name MUST begin with `x-`, for example, `x-internal-id`. The value can be `null`, a primitive, an array or an object. See [Specification Extensions](#specificationExtensions) for further details.
 
+Representations may take the form of a wildcard (`*`) to designate any `Content-Type`, or a regular expression for matching a specific type
```

Comment:
There was some confusion regarding us calling this `representations` vs `content` object and I have many similar updates in PR #721 .  I'm going to merge the two PRs as the "idea" has been agreed upon and then we can raise issues against the wording in the spec.  /cc @fehguy